src/stores/historyStore.ts

```
@@ -0,0 +1,113 @@
+  // Computed filtered entries
+  filteredEntries: HistoryEntry[]
+  
+  // Actions
+  addEntry: (numbers: NumberArray, quantity: number, maxValue: number) => void
+  setFilter: (filter: string) => void
+  clearHistory: () => void
+}
+
+// Create the store with Immer middleware
+export const useHistoryStore = create<HistoryState>()(
+  immer((set, get) => ({
```

ESLint

[error] 'get' is defined but never used.

(@typescript-eslint/no-unused-vars)